pydoc/developer/BENCHMARK.md

```diff
@@ -0,0 +1,470 @@
+# PyTest Benchmarks
+
+**Purpose**: Outline the general steps and options available for running the new PyTest benchmarks
+
+The PyTest benchmark structure allows the flexibility of running the entire suite, a single file, 
+or a single benchmark.
+
+## Running The Full Suite
+In most cases, running the full benchmark suite is desired. The simplest way to do this is to navigate into the 
+root-level of arkouda and run `make benchmark`
+
+This will run the entire benchmark suite with the following command:
+```commandline
+python3 -m pytest -c benchmark.ini --benchmark-autosave --benchmark-storage=file://benchmark_v2/.benchmarks
+```

```

Comment:
After this block, we should mention that the resulting JSON files will be located in the `arkouda/benchmarks_v2/.benchmarks` folder. It may be useful to identify how the files are named as well.

Reply:
Done